Formula/cadical.rb

```diff
@@ -0,0 +1,46 @@
+class Cadical < Formula
+  desc "Clean and efficient state-of-the-art SAT solver"
+  homepage "http://fmv.jku.at/cadical/"
+  url "https://github.com/arminbiere/cadical/archive/refs/tags/rel-1.4.0.tar.gz"
+  sha256 "b5990bea7aa5df6002b9bfe782323742d786565b9dfee37bbc0dd0f4b8e65141"
+  license "MIT"
+
+  def install
+    system "./configure"
+    chdir "build" do
+      system "make"
+      bin.install "cadical"
```

Comment:
Should we not install `mobical`?

https://github.com/arminbiere/cadical/blob/master/BUILD.md#cadical-build

Reply:
If/when you make further changes, squash them with your first commit, please. Thanks!

Reply:
I don't think that it makes sense to install `mobical`. It is tool for generating test cases for `cadical`'s API (see description [here](https://people.kth.se/~artho/papers/tap-2013.pdf)).